### PR TITLE
TextBox & Card

### PR DESCRIPTION
**Pull request by janiekyu**
Card component: 
* React: https://dev--designwithcompassion.netlify.app/storybook/react/?path=/docs/components-card--docs
* Core: https://dev--designwithcompassion.netlify.app/storybook/core/?path=/docs/components-card--docs

TextBox component: Can be used on their own or to apply CDS text styles to content inside slots (Styles will need to be updated based on designer's input. Will add unit tests once the styles have been finalized.)
* React: https://dev--designwithcompassion.netlify.app/storybook/react/?path=/docs/components-textbox--docs
* Core: https://dev--designwithcompassion.netlify.app/storybook/core/?path=/docs/components-textbox--docs

### CODE REVIEW

**Comment by janiekyu** [comment]
Snapshot diffs are expected as we added new components.

File: packages/core/src/components/Card/card.scss [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  &__image {
+    width: 100%;
+    height: auto;
+    object-fit: cover;
+    border-top-left-radius: var(--cds-card-border-radius);
+    border-top-right-radius: var(--cds-card-border-radius);
+  }
+
+  &__image--only {
+    border-radius: var(--cds-card-border-radius);
```
Comment (kendrick) [inline packages/core/src/components/Card/card.scss:19]: It doesn’t look like this value is applied to the image when only an `imagePath` is supplied—meaning only the top left/top right corners are rounded while the bottom are square:

image

File: packages/core/src/components/TextBox/textBox.scss [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+      font: var(--cds-type-heading-h6-desktop-bold);
+
+      &.cds--type__regularHeading {
+        font: var(--cds-type-heading-h6-desktop-regular);
+      }
+    }
+  }
+}
+
+.cds-text-box--mobile {
```
Comment (kendrick) [inline packages/core/src/components/TextBox/textBox.scss:176]: Happy to handle this with a separate issue/PR, but what are your thoughts on handling mobile/desktop sizes via media queries? (Perhaps only if the user opts in with a `.cds-textbox--responsive` class or something?)

File: packages/core/src/components/Card/card.scss (hunk 1 above)
Comment (janiekyu) [reply packages/core/src/components/Card/card.scss:19]: Thanks for catching this.

File: packages/core/src/components/TextBox/textBox.scss (hunk 2 above)
Comment (janiekyu) [reply packages/core/src/components/TextBox/textBox.scss:176]: That is a good idea. Let's follow up in a separate PR.

**Comment by kendrick** [comment]
Looks great! Merge away. 😄

**Pull request merged**